### DISCUSSION TITLE
Keep module imported in powershell core

### DISCUSSION THREAD

**Discussion by FLAMESpl** [PowerShell/PowerShell#17673] [Q&A] [answered]
Hello,

in windows powershell I had modules that was kept imported between terminal restarts, today I have tried powershell 7 and I have to import my modules every time I open new terminal. Is there a way to retain old functionality in new pwsh?

**Comment by doctordns** [comment, marked as answer]
If you place the module in one of the correct paths, module auto-load "should just work". 
What is the full path to your module?
With PowerShell 7, check the default module locations by running ` ($env:psmodulepath).split(';')`. On my host, the yields:
```powershell
PS C:\Foo> ($env:psmodulepath).split(';')
C:\Users\tfl.COOKHAM\Documents\PowerShell\Modules
C:\Program Files\PowerShell\Modules
c:\program files\powershell\7\Modules
C:\Program Files\WindowsPowerShell\Modules
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
```

You should have the module either under the first (the current user only) or the second *all users on this host).
